### DISCUSSION TITLE
Select Multiple Days (Not a Range)

### DISCUSSION THREAD

**Discussion by thekendog** [easepick/easepick#104] [General]
Is it currently possible to select multiple dates that aren't a range?

**Comment by mrpsiho** [comment]
I am looking forward for this feature too!

**Comment by wakirin** [comment]
this plugin in progress: https://github.com/easepick/easepick/tree/mselect-plugin
release date: undefined

**Comment by vk496** [comment]
Would love to see that happen!

**Comment by fredasterisk** [comment]
I am also looking forward to this feature! I'm currently using a multiselect input to store the selected dates but it's heavier than it should be!
